### DISCUSSION TITLE
Add downloadable file - extension

### DISCUSSION THREAD

**Discussion by DevBlokJochem** [Minestom/Minestom#1149] [Help] [answered]
Hello, I want to add a downloadable .jar file in my extension. Like you can download the file of immortal. (see the image)
![image](https://user-images.githubusercontent.com/71212639/171656414-30110bf0-3b12-4b55-baa3-5d7bcd6300ff.png)
How can I add/set the downloadable .jar?

**Comment by ZakShearman** [comment, marked as answer]
These JARs are automatically taken from the latest release for your project on GitHub.

You can see them under the `Assets` tab on other's releases: https://github.com/EmortalMC/Immortal/releases
